### PR TITLE
update using for mac

### LINKED ISSUE #60: Bug: Correct README to show correct directions for Mac, and how to set up virtual environment
**Description:**
The directions to run the project on the [README.md](README.md) file show a really simplistic set of steps for working with Python to run the app. These can be incorrect due to operating systems, multiple Python installs (2.x and 3.x), or some other factors. Some want to use virtual environments instead.

This issue should make more specific directions for operating systems as well as include virtual environments.

**Requirements:**
* Should have valid directions to run the app using Python 3 (probably with the `python3` command instead)
* Should offer a way to do virtual environments (`pyenv` or `venv` perhaps)
* Should include directions a bit more specific to Mac.
* Does _not_ need to tell how to install Python, but should link to places to install it (or just show something like `brew install python` or `apt-get install python3` or `choco install  python3` or other variations)
* _Does_ need to assume a user is not familiar with the install process of this app and try to make it as clear and friendly as possible

**Claiming This Issue:**
* Please leave a comment below indicating your interest. I can then assign it to you.
* If I don't see an update (comment, code, or PR), then I'll take your name off.
* You would always be welcome to reclaim it again.
* If you can't work on it, either leave a comment or remove yourself from the issue.
* If you have questions, please let me know!

### PR DESCRIPTION
**Pull request by IsabellaDev**
[issue60](https://github.com/codethesaurus/codethesaur.us/issues/60)  

I've updated the readme file to support mac installation.

If any changes, let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,16 +17,47 @@ Python 3.x installed, you may need to suffix all `python` and `pip` commands
...
+
+python manage.py runserver
+
+//  In your browser, visit http:localhost:8000
+//   CTRL - C in the terminal to stop the server
+
+// deactivate the virtual environment
+
+deactivate
+```
```
Comment (geekygirlsarah) [inline README.md]: These directions are great, but they'd be a bit easier to parse if it wasn't a large code block with directions and things to type in at the same time. Could you separate them out into list items like above (lines 23-29)?

File: README.md [hunk 2]
```diff
@@ -17,16 +17,47 @@ Python 3.x installed, you may need to suffix all `python` and `pip` commands
 with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 3 your default Python installation.
 
+
 ## Cloning and running it locally
 
-1. Clone the project (`git clone https://github.com/codethesaurus/codethesaur.us.git`)
+1. Clone the project (`C`)
```
Comment (geekygirlsarah) [inline README.md]: What happened to this command? What does the `C` mean?

File: README.md (hunk 1 above)
Comment (IsabellaDev) [reply README.md]: Hi I have updated it to keep format consistency, please check if it satisfies. Thanks!

File: README.md [hunk 3]
```diff
@@ -27,6 +28,18 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 1. In your browser, visit [http://127.0.0.1:8000/](http://127.0.0.1:8000/) or [http://localhost:8000/](http://localhost:8000/)
 1. Press CTRL+C in the terminal to stop the server.
 
+## For mac users
+
+1. Clone the project (`git clone https://github.com/codethesaurus/codethesaur.us.git`)
+1. Run `pip3 install virtualenv`
+1. To set up new virtual environment, run `virtualenv --no-site-packages venv`
+1. To activate virtual environment, run `source venv/bin/activate`
+1. Run `pip install -r requirements.txt`
```
Comment (geekygirlsarah) [inline README.md:37]: Change this to `pip3` and line 38 to `python3` and I'll merge it!

Comment (IsabellaDev) [reply README.md:37]: Actually, in the virtual environment, we don't need pip3 or python3, pip or python should be okay. But if you still want this change, I can make it too. Please let me know. Thanks.

Comment (geekygirlsarah) [reply README.md:37]: oh yeah, good point. Alright, let's do it!

**Pull request merged**